source/agora/crypto/Hash.d

```diff
@@ -101,7 +101,6 @@ private enum hasComputeHashMethod (T) = is(T == struct)
     Params:
       T = Type of struct to hash
       record = Instance of `T` to hash
-      state  = State delegate, when this struct is nested in another.
```

Comment:
The issue is that `state` was renamed to `hasher`. Just change the name to hasher and DDOC will pick it up.